lib/functions.php

```diff
@@ -197,6 +197,15 @@ function($matches) {
     if (!$result) {
         $result = _parse_fallback($uri);
     }
+    else
+    {
+        // Fix windows paths
+        if (isset($result['path']) && preg_match("#^(\w:)#", $result['path']))
```

Comment:
I worry that this change is a bit too aggressive. Do we really want to add a slash to every single uri that doesn't start with a slash? Probably not.

So if this is the case for `file://` uris (I think this is true) then this can be simplified significantly with:

```php
if (isset($result['scheme']) && $result['scheme']==='file' && $result['path'][0]!=='/') {
  $result['path'] = '/' + $result['path'];
}
```

Aside from that tweak, I would also say that it can use a better comment explaining exactly why this needs to be done, because I'm worried `fix windows paths` might not make sense to a developer maintaining this.